tests/Parse/MarkdownTest.php

```diff
@@ -20,42 +20,42 @@ class MarkdownTest extends TestCase
      */
     public function testParse($name, $markdown, $html)
     {
-        $this->assertEquals($html, $this->removeLineEndings(Markdown::parse($markdown)), 'Markdown test case "' . $name . '" failed');
+        $this->assertEquals($html, self::removeLineEndings(Markdown::parse($markdown)), 'Markdown test case "' . $name . '" failed');
```

Comment:
We prefer `static` over `self` for calls like this. Could you change it here and in the other spots in this file :)

```suggestion
        $this->assertEquals($html, static::removeLineEndings(Markdown::parse($markdown)), 'Markdown test case "' . $name . '" failed');
```

Reply:
As it turns out, the functions in MarkdownTest.php didn´t need to be changed into static functions. I´ve removed these changes